Source/Renderer/Texture.js

```diff
@@ -197,6 +197,15 @@ function Texture(options) {
 
   gl.pixelStorei(gl.UNPACK_ALIGNMENT, unpackAlignment);
 
+  if (options.skipColorSpaceConversion) {
+    gl.pixelStorei(gl.UNPACK_COLORSPACE_CONVERSION_WEBGL, gl.NONE);
+  } else {
+    gl.pixelStorei(
+      gl.UNPACK_COLORSPACE_CONVERSION_WEBGL,
+      gl.BROWSER_DEFAULT_WEBGL
+    );
+  }
```

Comment:
This same code should also go in `Texture.prototype.copyFrom`

Reply:
Just to clarify, in this function, I should change `options` to `source`, right?